terraform/fec_rds.tf

```diff
@@ -105,6 +105,11 @@ resource "aws_db_parameter_group" "fec_default" {
         name  = "log_statement"
         value = "ddl"
     }
+
+    parameter {
+        name  = "max_standby_streaming_delay" # This has no effect on masters, it only affects slaves
+        value = "1 h"
```

Comment:
From https://www.postgresql.org/docs/9.5/static/runtime-config-replication.html#GUC-MAX-STANDBY-STREAMING-DELAY, "_Units are milliseconds if not specified_". @LindsayYoung and I discussed the value and decided that `1h` was a reasonable tradeoff between performance and having data current.